imgui.h

```diff
@@ -267,6 +267,7 @@ struct ImVec2
 #ifdef IM_VEC2_CLASS_EXTRA
     IM_VEC2_CLASS_EXTRA     // Define additional constructors and implicit cast operators in imconfig.h to convert back and forth between your math types and ImVec2.
 #endif
+    inline ImVec2 & operator /=(ImVec2 other)  {x /= other.x; y /= other.y; return *this; }
```

Comment:
There are already math operators defined for `ImVec2`, they're just [disabled by default](https://github.com/ocornut/imgui/blob/240ab5890b2e8da294937a1710b021ac3f271472/imgui.h#L2479-L2507), and none of the backends enable them. Not sure if there's a reason for that or if it was just out of habit.

It might be easier to just match the existing backends and do the math manually per-component for consistency.

Reply:
They are disabled by default as I think it caused issues for users of `IM_VEC2_CLASS_EXTRA` with own math class + own math operators, though honestly I am not sure of specifics off-hand that that might need a separate investigation.

But yeah at the moment it'd be good if backends didn't rely on those.